Revit_Core_Engine/Revit_Core_Engine.csproj

```diff
@@ -124,6 +124,9 @@
     <AssemblyName>Revit_Core_Engine_2021</AssemblyName>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Units_Engine">
```

Comment:
There is also an issue with Units_Engine @tiagogrossi - not sure what's the problem as the code built from source without an issue for me.

The only thing I can think of is your new reference missing the `SpecificVersion == false` property.
      `<SpecificVersion>False</SpecificVersion>`

Reply:
I think it's more likely to be that the [dependencies file](https://github.com/BHoM/Revit_Toolkit/blob/master/dependencies.txt) does not say that `Revit_Toolkit` depends on `Localisation_Toolkit` in order to build, and as such, `check core` won't checkout `Localisation_Toolkit` to get the `Units_Engine` reference.

`Check installer` however of course checks all toolkits within the installer, which includes `Localisation_Toolkit` in advance of `Revit_Toolkit` and so that passes for build.

If this toolkit now depends on `Localisation_Toolkit` as a result of this PR, then the [dependencies file](https://github.com/BHoM/Revit_Toolkit/blob/master/dependencies.txt) should be updated in this PR as well.